.github/workflows/ruby.yml

```diff
@@ -86,5 +86,22 @@ jobs:
           bundler-cache: true
       - name: Rubocop run
         run: bundle exec rubocop
-
-    
+  traceroute:
+    needs: package-download
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        node: [14.18.1]
+        ruby: [2.7.4]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Run traceroute to verify unused routs
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - name: Traceroute run
+        run: rake traceroute
```

Comment:
Based on the error provided by the workflow action, I think this should be `bundle exec rake traceroute`.